demos/src/contrast-checker/contrast-checker.scss

```diff
@@ -0,0 +1,218 @@
+$o-colors-is-silent: false;
+
+// Import colors module
+@import '../../../main';
+
+@mixin swatchStyle() {
+	appearance: none;
+	border: 1px solid lightgrey;
+	height: 24px;
+	width: 24px;
+	cursor: pointer;
+
+	&:checked:not(.o-overlay-trigger) {
+		border: 2px solid deeppink;
+	}
+}
+
+:root {
+	--background: white;
+	--foreground: black;
+}
+
+body {
+	font-family: MetricWeb, sans-serif;
```

Comment:
This should be `Arial` for the whitelabel brand. When components don't rely on `o-typography` it's awkward to output the correct font family. For now I've been relying on a crude conditional e.g. https://github.com/Financial-Times/o-overlay/blob/master/demos/src/demo.scss#L5

We could tackle it by updating o-typography to make font families available with CSS custom properties.

Reply:
We agreed to leave it as is because this is a tool we want looking presentable rather than a component for a product to implement 👍